src/components/ObservingConditionsApplet.jsx

```diff
@@ -0,0 +1,519 @@
+import React from "react";
+import { Card, CardHeader, CardTitle, CardContent } from "@/components/ui/card";
+import {
+  Popover,
+  PopoverTrigger,
+  PopoverContent,
+} from "@/components/ui/popover";
+import DownloadIcon from "../assets/DownloadIcon.svg";
+import InfoIcon from "../assets/InfoIcon.svg";
+import { ChartContainer, ChartTooltip, ChartLegend } from "./ui/chart";
+import {
+  ComposedChart,
+  CartesianGrid,
+  XAxis,
+  YAxis,
+  Scatter,
+  Line,
+  Tooltip,
+  ResponsiveContainer,
+  ReferenceLine,
+  ZAxis,
```

Comment:
It looks like these 3 imports aren't being used:
```suggestion
  ReferenceLine,
```